Makefile

```diff
@@ -6,9 +6,8 @@ VERSION ?= 1.0.0
 EPOCH ?= 1
 MAINTAINER ?= "Community"
 
-CMDS = $(shell find ./cmd -maxdepth 1 -mindepth 1 -type d | grep -v testdata)
-CMD_BASENAMES = $(shell echo $(CMDS) | xargs -n1 basename)
-CMD_BINS = $(addprefix bin/, $(CMD_BASENAMES) )
+CMDS = admin boulder ceremony ct-test-srv
```

Comment:
I have two very vague concerns about explicit listing these four binaries here:
1. This is right at the top of the file, but ends up only being used by a few targets. In particular, "make build_cmds" will compile _everything_ in the repo, including all the other test services, not just these four.
2. It seems like it would be easy to create a new non-boulder executable and forget to add it to this list. Maybe that's fine -- this is intended to be the "what gets packaged" list -- but that circles back to my first point: this _looks_ like the "what gets built" list.

I don't know that there's anything super actionable in this feedback. Feel free to ignore it -- I think there's actually a larger cleanup to be done here that I can tackle after this merges (why do we define OBJECTS to be a copy of CMD_BINS? why do we disallow saying "make boulder" or "make admin"? why do we allow saying "make bin/boulder", but when you do, it builds all commands not just that one?).

Reply:
I definitely don't love listing them here, but I decided against trying to do a larger makefile refactor as that's some neurons not fully activated right now